compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Monoid.hs

```diff
@@ -23,38 +47,71 @@ newtype RenderEnv = RenderEnv (Set.Set Anchor)
 -- Another possibly reason is that the anchor refers to a definition in another
 -- package (and at the moment it's not possible to link accross packages).
 renderAnchorAvailable :: RenderEnv -> Anchor -> Bool
-renderAnchorAvailable (RenderEnv anchors) anchor = Set.member anchor anchors
+renderAnchorAvailable RenderEnv{..} anchor = isJust (lookupAnchor anchor)
```

Comment:
It looks like we aren’t using this anymore?

Reply:
Right! Thanks!